code-challenges.test.js

```diff
@@ -15,43 +15,144 @@
 // a) Create a test with an expect statement using the variable provided.
 // HINT: Check out this resource: https://jestjs.io/docs/expect#expectarraycontainingarray
 
-const colors1 = ["purple", "blue", "green", "yellow", "pink"]
+// describe("shuffleContent", () => {
+//   it("removes the first item from the array and shuffles the remaining content", () => {
+//     const colors1 = ["purple", "blue", "green", "yellow", "pink"]
+//     const colors2 = [
+//       "chartreuse",
+//       "indigo",
+//       "periwinkle",
+//       "ochre",
+//       "aquamarine",
+//       "saffron"
+//     ]
+//     expect(shuffleContent(colors1)).not.toEqual(expect.arrayContaining(expected))
+//     expect(shuffleContent(colors2)).not.toEqual(expect.arrayContaining(expected))
+//   })
+// })
+
+// const colors1 = ["purple", "blue", "green", "yellow", "pink"]
 // Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
-const colors2 = [
-  "chartreuse",
-  "indigo",
-  "periwinkle",
-  "ochre",
-  "aquamarine",
-  "saffron"
-]
+// const colors2 = [
+//   "chartreuse",
+//   "indigo",
+//   "periwinkle",
+//   "ochre",
+//   "aquamarine",
+//   "saffron"
+// ]
 // Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
+// FAIL  ./code-challenges.test.js
+// shuffleContent
+//   ✕ removes the first item from the array and shuffles the remaining content
+// ● shuffleContent › removes the first item from the array and shuffles the remaining content
+//   ReferenceError: shuffleContent is not defined
+
+// Pseudo code:
+// input: array
+// output: array with first item removed, then shuffles remaining items
+// process: 
+// iteration is needed, maybe for loop? or maybe .filter() ?
+// first remove first item of array, can use .shift() as it removes value at the first index.
+// Math.random() arrays index and not the value.
+
 
 // b) Create the function that makes the test pass.
 
-// Pseudo code:
+// const shuffleContent = (array) => {
+//   let newArr = []
+//   for ( let i=0; i < array.length; i++){
+//     newArr.shift(array[i].Math.random())
```

Comment:
It seems like you are trying to solve everything at once, which is confusing and overwhelming you. You are starting off on a good approach by first trying to remove the first item using .shift, but then the code is getting scrambled. Try to think of it step by step instead of trying to write all the logic required at once!